resources/Materials/TestSuite/stdlib/definition/definition_using_definitions.mtlx

```diff
@@ -0,0 +1,393 @@
+<?xml version="1.0"?>
```

Comment:
I see a few minor formatting issues in this new document, and I'd recommend running it through mxupdate.py before we merge it to main.

Reply:
Ran through mxupdate now.